system/Database/OCI8/Connection.php

```diff
@@ -53,7 +53,7 @@ class Connection extends BaseConnection
     protected $validDSNs = [
         'tns' => '/^\(DESCRIPTION=(\(.+\)){2,}\)$/', // TNS
         // Easy Connect string (Oracle 10g+)
-        'ec' => '/^(\/\/)?[a-z0-9.:_-]+(:[1-9][0-9]{0,4})?(\/[a-z0-9$_]+)?(:[^\/])?(\/[a-z0-9$_]+)?$/i',
+        'ec' => '/^(\/\/)?[a-z0-9.:_-]+(:[1-9]\d{0,4})?(\/[a-z0-9$_]+)?(:[^\/])?(\/[a-z0-9$_]+)?$/i',
```

Comment:
I propose to disable this rule.
This change is dangerous. The meanings of `[0-9]` and `\d` are different.
`\d` might match number characters like `０`, `६`, `೬` other than `0-9`.
See https://3v4l.org/icdLl